virtme/commands/run.py

```diff
@@ -1293,28 +1301,53 @@ def do_it() -> int:
     else:
         virtme_init_cmd = "virtme-init"
 
+    initsh = []
+    if args.systemd:
+        kernelargs.append("virtme_systemd=1")
+        kernelargs.append(
+            " ".join([f"console={console}" for console in arch.serial_console_args()])
+        )
+        initsh.extend(
+            [f"mount --bind /dev/null {unit}" for unit in get_conf("systemd.masks")]
```

Comment:
You may want to do:
```
 [f"mount --bind /dev/null {unit}" for unit in get_conf("systemd.masks") or []]
```

Otherwise if `systemd.masks` isn't defined we get a `'NoneType' object is not iterable` error.

Reply:
Fixed in https://github.com/arighi/virtme-ng/pull/280/commits/ca98cdc43f8bc3442dc1e34d71c09982ea585834!

Thanks @arighi I'll try it with Ubuntu|Arch next :)